package.json

```diff
@@ -1,7 +1,7 @@
 {
   "private": true,
   "scripts": {
-    "install": "lerna bootstrap --hoist",
+    "install": "npm run update:core && lerna bootstrap --hoist",
```

Comment:
The workflow would be to `npm install` at the top level, then run `add-sibling`.